support-frontend/assets/helpers/tracking/googleTagManager.js

```diff
@@ -277,16 +295,22 @@ function init(participations: Participations) {
   onConsentChangeEvent((thirdPartyTrackingConsent: {
     [key: string]: boolean
   }) => {
+
+    // Update vendorConsentsLookup value based on thirdPartyTrackingConsent
+    vendorConsentsLookup = Object.keys(thirdPartyTrackingConsent).reduce((accumulator, vendorKey) => {
+      if (thirdPartyTrackingConsent[vendorKey]) {
+        return [...accumulator, vendorKey];
+      }
+      return [...accumulator];
+    }, []).join(',');
```

Comment:
Is [.filter()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) supported? e.g

```
vendorConsentsLookup = Object.keys(thirdPartyTrackingConsent).filter((vendorKey) => thirdPartyTrackingConsent[vendorKey]).join(',')
```

Reply:
Yes it is, this is a nice suggestion, happy to adopt this 👍